js/ez-navigation-hub.js

```diff
@@ -1,5 +1,33 @@
 (function () {
-    /** @polymerElement */
+    /**
+    * <ez-navigation-hub> represents the navigation hub of the application.
+    * Zones can be selected to display the links refering to the different parts of the application.
+    *
+    * Each zone of the navigation hub needs a data-zone-identifier.
+    * Example :
+    *
+    *    <ul>
+    *      <li data-zone-identifier="1">
+    *        <a href="#">zone1</a>
+    *      </li>
+    *    </ul>
+    *
+    * Each link releated to a zone must contain a data-zone with the matching identifier.
```

Comment:
To appear in a zone, a link should have a `data-zone` attribute containing the corresponding zone identifier. In addition, such links should have the class `ez-zone-link`.